.github/ISSUE_TEMPLATE/member_build_cache_issue.md

```diff
@@ -0,0 +1,11 @@
+---
+name: 'Member: Issue for the `build-cache` team'
+about: 'Report a problem about the Gradle build-cache'
+labels: '@build-cache, from:member'
+milestone: '5.6 RC1'
```

Comment:
this works via url, but not as a template, e.g.:
https://github.com/gradle/gradle/issues/new?labels=%40build-cache%2C+from%3Amember&milestone=5.6%20RC1&assignee=paplorinc&body=%3C---TODO--%3E&title=TODO